src/python/ensembl/utils/rloader.py

```diff
@@ -50,21 +50,20 @@ def __init__(self, parser: Optional[str] = None) -> None:
     def __parse(self, content: str) -> Any:
         if self.parser == 'yaml':
             return yaml.load(content, yaml.SafeLoader)
-        elif self.parser == 'ini':
+        if self.parser == 'ini':
             config = configparser.ConfigParser()
             try:
                 config.read_string(content)
             except configparser.MissingSectionHeaderError:
                 content = "[DEFAULT]\n" + content
                 config.read_string(content)
             return config
-        elif self.parser == 'env':
+        if self.parser == 'env':
```

Comment:
why changing that? (just for my knowledge)

Reply:
`pylint` claims (accurately, I would say) that since the previous `if` will always end up in the `return` command (or raising an exception), you will never see run the code for both conditions.

I can see benefits of both approaches, with `elif` providing slightly better readability, so we could decide to ignore this error if you prefer.

Reply:
No problem, was just curious